bin/signalk-server

```diff
@@ -16,22 +16,6 @@
  * limitations under the License.
 */
 
-const { engines } = require('../package.json')
-const semver = require('semver')
-const minimumVersion = engines.node
-const recommendedVersion = '20 - 22'
-
-if (!semver.satisfies(process.version, minimumVersion)) {
-  console.error(
-    `The installed version of node (${process.version}) is older than the minimum required version (${minimumVersion}). See https://github.com/SignalK/signalk-server/wiki/Installing-and-Updating-Node.js for more information how to upgrade.`
-  )
-  process.exit(1)
-} else if (!semver.satisfies(process.version, recommendedVersion)) {
-  console.warn(
-    `The installed version of node (${process.version}) is different than the recommended version (${recommendedVersion}). See https://github.com/SignalK/signalk-server/wiki/Installing-and-Updating-Node.js for more information how to upgrade.`
-  )
-}
-
```

Comment:
I moved this into src/version.ts just so it's type checked, and it's now called when `Server` is initialized.